torch/nn/_functions/bilinear.py

```diff
@@ -0,0 +1,66 @@
+import torch
+from torch.autograd import Function
+
+
+class Bilinear(Function):
+
+    def forward(self, input1, input2, weight, bias=None):
+        self.save_for_backward(input1, input2, weight, bias)
+
+        output = input1.new()
+
+        self.buff2 = input1.new()
```

Comment:
Correct me if I'm wrong, but we are not reusing the value of `buff2` in backward, so you can define a `buff2` in backward again and it should be fine